assembl/static2/js/app/components/cookies/cookieSetter.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow
+/* global globalAnalytics */
+import React from 'react';
+import { I18n } from 'react-redux-i18n';
+import classnames from 'classnames';
+
+// Components
+import Helper from '../common/helper';
+import CookieSwitch from './cookieSwitch';
+
+import { COOKIE_TRANSLATION_KEYS } from '../../constants';
+
+export type CookieObject = {
```

Comment:
Could it be worth to put in flow/object_types.js if used somewhere else ?

Reply:
It's only used in this folder